plugins/UnrealAgent/AGENTS.md

```
@@ -0,0 +1,58 @@
+# Plugins/UnrealAgent
+
+Editor-only Unreal plugin for an in-editor OpenCode ACP assistant panel. This is separate from `plugins/McpAutomationBridge`; it does not expose MCP tools itself, but it can pass the configured `unreal-engine` MCP server to OpenCode ACP sessions.
+
+## STRUCTURE
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)